Python/Pangram_in_python.py

```diff
@@ -0,0 +1,34 @@
+# Python3 code to check if the given string is a Pangram 
+
+import string  //Using the string on your sample code
+  
+def ispangram(str): 
+    alphabet = "abcdefghijklmnopqrstuvwxyz"
+    
+    for char in alphabet: 
+        if char not in str.lower(): 
+            return False
+  
+return True
+      
+      
+# Sample code 
+string = 'the quick brown fox jumps over the lazy dog'
+if(ispangram(string) == True): 
+    print("Given String is a Pangram") 
+else: 
+    print("Given String is not Pangram") 
+    
+    
+/*    
```

Comment:
Make `/*` to `'''`
It will then be a multiline comment in python.
Similarly `*/` to `'''`

Reply:
done!